src/main/java/se/nackademin/Inventory.java

```diff
@@ -0,0 +1,80 @@
+package se.nackademin;
+
+import java.util.HashMap;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Inventory {
+    private String vendor;
+    private boolean vendorExists;
+    public String productName;
+    private Boolean inStock;
+    private int amount;
+    public HashMap<String, Integer> productMap = new HashMap<String, Integer>();
+    public List<String> vendorList = new ArrayList<>();
+
+    public Inventory(String vendor, String productName, Boolean inStock, Boolean vendorExists, int amount) {
+        this.vendor = vendor;
+        this.vendorExists = vendorExists;
+        this.productName = productName;
+        this.inStock = inStock;
+        this.amount = amount;
+
+        vendorList.add(0, this.vendor);
+        productMap.put(this.productName, this.amount);
+    }
+
+    public Inventory() {}
+
+    public String setNewVendor(String vendor) {
+        this.vendor = vendor;
+        vendorList.add(0, this.vendor);
+        return this.vendor;
+    }
+
+    public String getVendorName(int index) {
+        String element = vendorList.get(index);
+        return element;
+    }
+
+    public Boolean checkVendorAvailability(String element) {
+        if (element.isBlank()) {
```

Comment:
Compile error, should be `isEmpty`. isBlank would work if the source leverl in pom.xml was set to 11 or higher.